src/models/zone.cr

```diff
@@ -14,7 +14,7 @@ module PlaceOS::Model
 
     attribute name : String, es_type: "keyword"
     attribute description : String
-    attribute tags : String
+    attribute tags : Array(String) = ->{ [] of String }
```

Comment:
Rather than being an array, can it be a expressed as a `Set(String)` (if supported)?

Reply:
depends on whether the rethinkdb driver has implemented `Set` serialisation (which i doubt)

Reply:
Doesn't look like it.
I definitely agree with attempting to encode properties into the DB types as much as possible.
Shouldn't be too hard if you want to open a PR on [crystal-rethinkdb](https://github.com/kingsleyh/crystal-rethinkdb).

That driver could use a bit of loving.

Reply:
We can ensure uniqueness at the ORM level

Reply:
I think having a type-level restriction would be nicer

```crystal
a = %w(hello hello hello bye good).to_json # => ["hello","hello","hello","bye","good"]
Set(String).from_json(a).to_json # => ["hello","bye","good"]
```

Reply:
tried it now! looks like it works! @KimBurgess